_search-plugins/querqy/index.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+title: Querqy plugin
```

Comment:
Change title to just "Querqy" to be consistent with the other plugin docs (like the SQL plugin page is just "SQL" and not "SQL plugin")

Reply:
done